packages/runtime/runtime-definitions/src/dataStoreRegistry.ts

```diff
@@ -20,7 +20,8 @@ export type FluidDataStoreRegistryEntry =
  * An associated pair of an identifier and registry entry.  Registry entries
  * may be dynamically loaded.
  */
-export type NamedFluidDataStoreRegistryEntry = [string, Promise<FluidDataStoreRegistryEntry>];
+export type NamedFluidDataStoreRegistryEntry =
```

Comment:
is this change used anywhere here/in the referenced draft pr?

Reply:
Removing, it's not relates to this change